libs/api/domains/auth/src/lib/services-v2/meDelegations.service.ts

```diff
@@ -190,12 +187,15 @@ export class MeDelegationsServiceV2 implements MeDelegationsServiceInterface {
     if (!delegation) {
       return delegation
     }
+
     return {
       ...delegation,
-      scopes: delegation.scopes?.map((scope) => ({
-        ...scope,
-        domainName: delegation.domainName,
-      })),
+      scopes: delegation.scopes
+        ?.filter((scope) => scope?.validTo && scope.validTo > new Date())
```

Comment:
We also need to show scopes which are valid to and including "today". These are saved as the start of today, so maybe use date-fns `const today = startOfDay(new Date())` and check if `scope.validTo >= today`.